internal/pkg/agent/cmd/install.go

```diff
@@ -59,6 +60,11 @@ func installCmd(streams *cli.IOStreams, cmd *cobra.Command, args []string) error
 		return fmt.Errorf("already installed at: %s", paths.InstallPath)
 	}
 
+	nonInteractive, _ := cmd.Flags().GetBool("non-interactive")
+	if nonInteractive {
+		fmt.Fprintf(streams.Out, "installing in non-interactive mode")
```

Comment:
Seems that most outputs here start with a capital letter and end in a period. Would be better to change that to match.

```suggestion
		fmt.Fprintf(streams.Out, "Installing in non-interactive mode.")
```